scss/product/components/_forms.scss

```diff
@@ -368,8 +369,9 @@ select.form-control-lg {
     // Allow folks to *not* use `.form-group`
     .form-control {
       display: inline-block;
-      width: auto; // Prevent labels from stacking above inputs in `.form-group`
+      width: 80px;
       vertical-align: middle;
+      margin: 0 7px;
```

Comment:
why fixed width

Reply:
Because by default width of the input is much wider than what is required for inline input. Please find the screenshot below
![Screenshot from 2020-03-11 22-08-50](https://user-images.githubusercontent.com/34312966/76441263-23f2d980-63e5-11ea-929a-ca5a1f809c14.png)

Reply:
this will break the input field in other place for sm screens. for inline usage we can wrap it inside a div with fixed width 

eg

```
<div class="w-25">
  <input type="text">
</div>
```

Reply:
The above approach doesn't work. 
![Screenshot from 2020-03-16 12-10-54](https://user-images.githubusercontent.com/34312966/76729485-4ab36600-677f-11ea-8483-081fcc70029a.png), even when you reduce the width of div, it doesn't decrease the width. It considers the width of input field

Reply:
@visualapps `flex-shrink-*` will not apply on input size. It takes the default input size. On the contrary, `flex-grow-*` works because it is more than the default width. In order to have a lesser width for the input field, we have to decrease the default input width.
![Screenshot from 2020-03-17 20-04-50](https://user-images.githubusercontent.com/34312966/76867620-cbf81f00-688b-11ea-92fd-655463100e84.png)

And any input width does not change based on its content size. 
There is one more way we can use `flex-basis`. But even then we will have to define definite width. https://gedd.ski/post/the-difference-between-width-and-flex-basis/

Reply:
can we have a class such as, `form-control-inline` 

```
.form-control-inline{
 display:inline
 width: auto;
}
```

`<input type="text" class="form-control from-control-inline">
`

Reply:
Or we need to have various  width classes. similar to this we can use this to any element
https://tailwindcss.com/docs/width#fixed-width
<img width="714" alt="Screen Shot 2020-03-19 at 2 43 12 PM" src="https://user-images.githubusercontent.com/18216179/77050648-15617f00-69f0-11ea-98c9-96d8562026e7.png">

Reply:
this will give us flexibility and allow to add width as class in any elements